tests/e2e/framework/kubernetes.go

```diff
@@ -605,7 +628,16 @@ func (k *KubeInfo) Teardown() error {
 	validatingWebhookConfigurationExists := false
 	log.Infof("Deleting namespace %v", k.Namespace)
 	for attempts := 1; attempts <= maxAttempts; attempts++ {
-		namespaceDeleted, _ = util.NamespaceDeleted(k.Namespace, k.KubeConfig)
+		if *useOperator {
+			if _, err := util.Shell("kubectl delete ns istio-operator --grace-period=0 --force",
```

Comment:
k.namespace and k.kubeconfig not passed in

Reply:
k.namesapace is in this case `istio-system`.  k.kubeconfig in this case maps to `k.namespace`.  There is no context for these, hen ce why they are not passed into the `shell` util command since they are not absolutely needed.

Reply:
but you are deleting istio-operator ns here? also why we don't remove the extra args if we don't pass in.

Reply:
why force and no grace period?

Reply:
In the non operator code this is checking if the namespace was deleted, but here we are deleting it. Don't we already delete it above?

Reply:
I have removed force and grace period.  The logic sets namespaceDeleted=true only if the namespace was deleted.  The same logic checks that the namespace is deleted:

```
                if namespaceDeleted && !validatingWebhookConfigurationExists {
                        break
                }       
```
This basically just adds logic for the case the --use_operator is set.

Reply:
> In the non operator code this is checking if the namespace was deleted, but here we are deleting it. Don't we already delete it above?

I see, I misunderstood the purpose of the function call.  I will check the logic again and submit a followup.